runtime/compiler/control/J9Options.cpp

```diff
@@ -2004,8 +2004,8 @@ bool J9::Options::preProcessCodeCacheXlpCodeCache(J9JavaVM *vm, J9JITConfig *jit
             {
             // Generate warning message for user that requested page sizes / type is not supported.
             char *oldQualifier, *newQualifier;
-            char *oldPageType = NULL;
-            char *newPageType = NULL;
+            const char *oldPageType;
+            const char *newPageType;
```

Comment:
Initialization of these variables was removed.  Looks like they might never be initialized before being used.